### ISSUE TITLE
wrong bgp6_session_prefix_count

### ISSUE THREAD

**Issue by skorpy2009** [czerwonk/bird_exporter#7]
The Exporter gives me: 
```
bgp6_session_prefix_count_export{name="netlab"} 0
bgp6_session_prefix_count_filter{name="netlab"} 0
bgp6_session_prefix_count_import{name="netlab"} 1
```

but bird6 gives me:
```
netlab   BGP      main_rib up     1503219650  Established
  Preference:     100
  Input filter:   internal_in
  Output filter:  internal_out
  Routes:         11 imported, 9 exported, 11 preferred
  Route change stats:     received   rejected   filtered    ignored   accepted
    Import updates:             31          0          0          1         30
    Import withdraws:           19          0        ---          0         19
    Export updates:            166        125          2        ---         39
    Export withdraws:           50        ---        ---        ---         29
  BGP state:          Established
```

**Comment by czerwonk** [comment]
Hi, I created a working branch (bug/issue7) and added a test for your output. The parser seems to work like expected. Do you use the latest version of bird_exporter? Can you please also add the parameters you are using to start the binary?

># go test -v
>=== RUN   TestEstablishedBgpOldTimeFormat
>--- PASS: TestEstablishedBgpOldTimeFormat (0.00s)
>=== RUN   TestEstablishedBgpCurrentTimeFormat
>--- PASS: TestEstablishedBgpCurrentTimeFormat (0.00s)
>=== RUN   TestIpv6Bgp
>--- PASS: TestIpv6Bgp (0.00s)
>=== RUN   TestActiveBgp
>--- PASS: TestActiveBgp (0.00s)
>=== RUN   Test2BgpSessions
>--- PASS: Test2BgpSessions (0.00s)
>=== RUN   TestOspfOldTimeFormat
>--- PASS: TestOspfOldTimeFormat (0.00s)
>=== RUN   TestOspfCurrentTimeFormat
>--- PASS: TestOspfCurrentTimeFormat (0.00s)
>=== RUN   TestOspfProtocolDown
>--- PASS: TestOspfProtocolDown (0.00s)
>=== RUN   TestOspfRunning
>--- PASS: TestOspfRunning (0.00s)
>=== RUN   TestOspfAlone
>--- PASS: TestOspfAlone (0.00s)
>=== RUN   TestIssue7
>--- PASS: TestIssue7 (0.00s)
>PASS
>ok  	github.com/czerwonk/bird_exporter	0.003s

**Comment by skorpy2009** [comment]
This here is the config and details of another server, having the same Problem

cat /etc/systemd/system/bird-exporter.service
```
[Unit]
Description=Bird Exporter
After=network.target

[Service]
User=bird
ExecStart=/usr/local/go/bin/bird_exporter -bird.socket /var/run/bird/bird.ctl -bird.socket6 /var/run/bird/bird6.ctl

[Install]
WantedBy=multi-user.target
```

```
regio_1  BGP      main_rib up     1505322632  Established
  Preference:     100
  Input filter:   metro_transit_in
  Output filter:  metro_transit_out
  Routes:         40304 imported, 3 exported, 80608 preferred
  Route change stats:     received   rejected   filtered    ignored   accepted
    Import updates:          90663          0         46      13683      76934
    Import withdraws:         7161          0        ---         46       7161
    Export updates:         117482     117197        271        ---         14
    Export withdraws:         7163        ---        ---        ---      40273
  BGP state:          Established
    Neighbor address: 2a02:648:ffff::e
    Neighbor AS:      8804
    Neighbor ID:      193.238.196.1
    Neighbor caps:    refresh enhanced-refresh AS4
    Session:          external AS4
    Source address:   2a02:648:ffff::d
    Hold timer:       127/180
    Keepalive timer:  7/60
```

```
# HELP bgp6_session_prefix_count_export Number of exported routes
# TYPE bgp6_session_prefix_count_export gauge
bgp6_session_prefix_count_export{name="access_01"} 14
bgp6_session_prefix_count_export{name="angela_off_aixit"} 0
bgp6_session_prefix_count_export{name="core1_off_aixit"} 14
bgp6_session_prefix_count_export{name="gw02_babel"} 14
bgp6_session_prefix_count_export{name="regio_1"} 0
# HELP bgp6_session_prefix_count_filter Number of filtered routes
# TYPE bgp6_session_prefix_count_filter gauge
bgp6_session_prefix_count_filter{name="access_01"} 0
bgp6_session_prefix_count_filter{name="angela_off_aixit"} 0
bgp6_session_prefix_count_filter{name="core1_off_aixit"} 0
bgp6_session_prefix_count_filter{name="gw02_babel"} 0
bgp6_session_prefix_count_filter{name="regio_1"} 0
# HELP bgp6_session_prefix_count_import Number of imported routes
# TYPE bgp6_session_prefix_count_import gauge
bgp6_session_prefix_count_import{name="access_01"} 12
bgp6_session_prefix_count_import{name="angela_off_aixit"} 0
bgp6_session_prefix_count_import{name="core1_off_aixit"} 15
bgp6_session_prefix_count_import{name="gw02_babel"} 12
bgp6_session_prefix_count_import{name="regio_1"} 3
```

**Comment by skorpy2009** [comment]
There could be a correlation to the fact, that this bird4 is doing nothing and only bird6 has BGP sessions...

**Comment by czerwonk** [comment]
Hi, i found the bug and fixed it in 0.8.3. Could you please test the newest release?

**Comment by skorpy2009** [comment]
Thank you, seems to work now!
